src/js/plugins/plugin.vendor-fullscreen.js

```diff
@@ -0,0 +1,247 @@
+    document.msFullscreenElement;
+}
+
+/**
+ * Returns true if the document is in fullscreen mode.
+ *
+ * @returns {boolean}
+ */
+export function isFullscreenActive() {
+  const fullscreenElement = this.getFullscreenElement();
+  return fullscreenElement !== null && fullscreenElement !== undefined;
+}
+
+/**
+ * Exits fullscreen mode.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Document/exitFullscreen
+ */
+export function exitFullscreen() {
+  if (document.exitFullscreen) {
+    document.exitFullscreen();
+  } else if (document.webkitExitFullscreen) {
+    document.webkitExitFullscreen();
+  } else if (document.mozCancelFullScreen) {
+    document.mozCancelFullScreen();
+  } else if (document.msExitFullscreen) {
+    document.msExitFullscreen();
+  }
+}
+
+/**
+ * Requests fullscreen mode for the given element
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen
+ */
+export function requestFullscreen(element) {
+  if (element.requestFullscreen) {
+    element.requestFullscreen();
+  } else if (element.webkitRequestFullscreen) {
+    element.webkitRequestFullscreen();
+  } else if (element.mozRequestFullScreen) {
+    element.mozRequestFullScreen();
+  } else if (element.msRequestFullscreen) {
+    element.msRequestFullscreen();
+  }
+}
+
+/**
+ * Returns true if fullscreen mode is allowed on this device and document.
+ * 
+ * @returns {boolean}
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenEnabled
+ */
+export function fullscreenAllowed() {
+  return (document.fullscreenEnabled ||
+    document.webkitFullscreenEnabled ||
+    document.mozFullScreenEnabled ||
+    document.msFullScreenEnabled);
+}
+
+/**
+ * jQuery-style binding to a fullscreenchange event.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenchange_event
+ */
+export function bindFullscreenChangeListener(
+  data, fullscreenchangeListener
+) {
+  const event = 'fullscreenchange ';
+  const vendor_prefixes = [
+    'webkit',
+    'moz',
+    'ms'
+  ];
+  const all_events = $.trim(event + vendor_prefixes.join(event) + event);
+  $(document).bind(all_events, data, fullscreenchangeListener);
+}
+
+/**
+ * Returns true if current device is mobile
+ *
+ * @returns {boolean}
+ */
+export function isMobile() {
+  return (typeof window.orientation !== 'undefined') || (navigator.userAgent.indexOf('IEMobile') !== -1);
+}
+
+/** @deprecated */
+BookReader.util.isMobile = isMobile;
+
+/** @deprecated */
+BookReader.util.getFullscreenElement = getFullscreenElement;
+
+/** @deprecated */
+BookReader.util.bindFullscreenChangeListener = bindFullscreenChangeListener;
+
+/** @deprecated */
+BookReader.util.fullscreenAllowed = fullscreenAllowed;
+
+/** @deprecated */
+BookReader.util.requestFullscreen = requestFullscreen;
+
+/** @deprecated */
+BookReader.util.exitFullscreen = exitFullscreen;
+
+/** @deprecated */
+BookReader.util.isFullscreenActive = isFullscreenActive;
```

Comment:
Let's place these inside the `if` statement, so that we don't modify `BookReader` at all unless necessary (as before)